docs/manual/about/release-process.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Release process
+description: Vector's release process. Covering cadence and compatibility guarantees.
+---
+
+Here you can find information about how to track Vector's releases, how often
+we release, compatibility guarantees, and more.
+
+## How to track releases
+
+There are a few different ways to track when there is a new release:
+
+* Follow [@vectordotdev](https://twitter.com/vectordotdev) on Twitter.
+* Heading to our [Discord server](https://discord.gg/EXXRbYCq) and watching the
+	`#announcements` channel.
+* Going to our [Github repository](https://github.com/timberio/vector) and
+	using the "watch" feature. Click "Custom" and then "Releases" to only be
+	notified for new releases.
+* If you are using [one of our package
+	repositories](https://cloudsmith.io/~timber/repos/vector/packages/) you should
+	be able to see the update available when updating your package lists.
+
+## Release cadence
+
+We aim to:
+
+* Release a stable version of Vector every 6 weeks.
+* Release patch fixes as needed to fix high-priority bugs and regressions.
+* Release daily builds representing the latest state of Vector for feedback.
```

Comment:
Should we call these "nightly builds" instead of "daily builds" for consistency?